messages/pt.json

```diff
@@ -1,5 +1,9 @@
 {
-  "admin/messages.headerTitle": "Tradução de produtos",
+  "admin/messages.header.product.title": "Traduções de produtos",
+  "admin/messages.header.sku.title": "Traduções de SKU",
+  "admin/messages.header.brand.title": "Traduções de marcas",
```

Comment:
`marca` or `marcas`? The Spanish version is using the singular form, while this is using the plural one.

Reply:
They all should be in singular form. Thanks!